src/Microsoft.AspNet.SignalR.Core/Configuration/DefaultConfigurationManager.cs

```diff
@@ -14,12 +14,15 @@ public class DefaultConfigurationManager : IConfigurationManager
         // if _minimumKeepAlivesPerDisconnectTimeout != 3, update the ArguementOutOfRanceExceptionMessage below
         private const int _minimumKeepAlivesPerDisconnectTimeout = 3;
 
+        internal const int DefaultMaxScaleoutMappingsPerStream = 1000000;
```

Comment:
Sounds good to me.

The basic idea is that it should be greater than the total number of messages sent through the backplane to anyone (not just the reconnecting client) in the time it takes for a client to reconnect should be less than the new number. I think that anything over 1000 is probably good enough for most cases. 65535 is probably safer, and now it's configurable, so if there's problems it can always be adjusted.